src/QMCHamiltonians/GridExternalPotential.cpp

```diff
@@ -18,6 +18,16 @@
 
 namespace qmcplusplus
 {
+GridExternalPotential::GridExternalPotential(ParticleSet& P) : ps_(P)
+{
+  setEnergyDomain(POTENTIAL);
+  oneBodyQuantumDomain(P);
+}
+
+std::string GridExternalPotential::getClassName() const { return "GridExternalPotential"; }
```

Comment:
Please keep this defined in the header file.

Reply:
done